podcasts/New Search/Views/History/SearchHistoryCell.swift

```diff
@@ -53,15 +53,18 @@ struct SearchHistoryCell: View {
                         let uuid = entry.podcast?.uuid ?? entry.episode?.podcastUuid {
                         if entry.podcast?.kind == .folder {
                             SearchFolderPreviewWrapper(uuid: uuid)
-                                .modifier(NormalCoverShadow())
-                                .frame(width: 48, height: 48)
+                                .cornerRadius(4)
+                                .shadow(radius: 3, x: 0, y: 1)
+                                .frame(width: 56, height: 56)
                                 .allowsHitTesting(false)
                                 .padding(.trailing, 12)
                         } else {
-                            PodcastCover(podcastUuid: uuid)
-                                .frame(width: 48, height: 48)
+                            PodcastImage(uuid: uuid)
+                                .cornerRadius(4)
+                                .frame(width: 56, height: 56)
                                 .allowsHitTesting(false)
                                 .padding(.trailing, 12)
+                                .shadow(radius: 3, x: 0, y: 1)
                         }
```

Comment:
Since this code is duplicated in a few places I wonder if we could make a common component so we don't need to change the values in 4 places?

```
struct SearchEntryImage: View {
    let uuid: String
    let kind: PodcastFolderSearchResult.Kind?
    
    var body: some View {
        image
            .frame(width: 56, height: 56)
            .cornerRadius(4)
            .shadow(radius: 3, x: 0, y: 1)
            .allowsHitTesting(false)
    }
    
    @ViewBuilder
    private var image: some View {
        if kind == .folder {
            SearchFolderPreviewWrapper(uuid: uuid)
        } else {
            PodcastImage(uuid: uuid)
        }
    }
}
```

Then here we just need to do:

```
                        SearchEntryImage(uuid: uuid, kind: entry.podcast?.kind)
                            .padding(.trailing, 12)
```